Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 ARG OZONE_RUNNER_IMAGE=apache/ozone-runner
-FROM ${OZONE_RUNNER_IMAGE}:20241108-jdk17-1
+FROM ${OZONE_RUNNER_IMAGE}:20241212-1-jdk11
```

Comment:
Should we use the latest jdk11 ozone-runner  "apache/ozone-runner:20241216-1-jdk11" ?

Reply:
OK, make sense.